deploy/kubernetes/log-server-service.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -v --file ../../docker-compose.yml
+    kompose.version: 1.4.0 (HEAD)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: log-server
+  name: log-server
+spec:
+  ports:
+  - name: "8090"
```

Comment:
looks odd - why name the port with a stringified number?  can't you give it a more meaningful label?

Reply:
This file has been generated from docker-compose.yml, which doesn't allow ports to be given names unfortunately.

Reply:
This file has been generated from docker-compose.yml, which doesn't allow ports to be given names unfortunately.